scripts/pepper

```diff
@@ -87,6 +88,8 @@ class Pepper(object):
                             )
                 else:
                     print(result)
+                if exit_code is None or exit_code == 0:
+                    exit_code = PepperRetcode().validate(self.cli.options, json.loads(result)['return'])
```

Comment:
Instead of validating any of this, I would just search for all the retcodes and return the largest number.

This will remove the need to have this extra retcode thing.